Index: open-axiom/src/sman/sman.c
===================================================================
--- open-axiom.orig/src/sman/sman.c	2013-09-01 11:22:27.000000000 +0400
+++ open-axiom/src/sman/sman.c	2013-09-01 11:22:27.000000000 +0400
@@ -449,7 +449,6 @@
   const char* command = oa_concatenate_string(root_dir,HypertexProgram);
 
   if (readablep(command) != 1) {
-    fprintf(stderr, "Hypertex program not found.\n");
     return;
   } 
   if (PasteFile){
@@ -479,8 +478,6 @@
 
   if (readablep(command) == 1)
     spawn_of_hell(command, DoItAgain);
-  else
-    fprintf(stderr, "Graphics program not found.\n");
 }
 
 /* Start the core executable session in a separate process, */
