Description: make /usr/bin/open-axiom a dynamically linked executable
Origin: vendor
Forwarded: no
Index: b/src/driver/Makefile.in
===================================================================
--- a/src/driver/Makefile.in	2013-05-09 20:02:37.000000000 +0400
+++ b/src/driver/Makefile.in	2013-05-09 20:02:43.000000000 +0400
@@ -70,7 +70,7 @@
 main.lo: open-axiom.h
 
 open-axiom$(EXEEXT): $(open_axiom_objects)
-	$(CXXLINK) -all-static -o $@ $(open_axiom_objects) $(open_axiom_LDADD)
+	$(CXXLINK) -o $@ $(open_axiom_objects) $(open_axiom_LDADD)
 
 mostlyclean-local:
 	@rm -f $(open_axiom_objects)
